src/main/java/com/google/api/codegen/PageStreamingConfig.java

```diff
@@ -77,20 +91,28 @@ public static PageStreamingConfig createPageStreaming(
               resourcesFieldName));
     }
 
-    if (requestTokenField == null || responseTokenField == null || resourcesField == null) {
+    if (requestPageSizeField == null || requestTokenField == null || responseTokenField == null || resourcesField == null) {
```

Comment:
This is a good point. None of the page-related methods of an iterator should exist unless the List request/response protos have all three pagination fields with the right types.